internal/drivers/sqlserver/sql.go

```diff
@@ -228,7 +228,7 @@ func parseURLToDSN(urlstr string) (string, error) {
 	}
 
 	if encoded := vals.Encode(); encoded != "" {
-		dsn.WriteString("?")
+		dsn.WriteString("/")
```

Comment:
shouldn't this be a query string and if so it will automatically append a ?.  probably need a test case for this

Reply:
Oops yeah you're right! I mis-read your older PR comment, but there's a test case that looks at this